src/main/features/claimant-response/helpers/taskListBuilder.ts

```diff
@@ -200,7 +200,7 @@ export class TaskListBuilder {
       )
       this.buildProposeAlternateRepaymentPlanTask(draft, tasks, externalId)
 
-      if (!claim.claimData.defendant.isBusiness()) {
+      if (!claim.claimData.defendant.isBusiness() || draft.acceptPaymentMethod.accept.option === YesNoOption.YES) {
```

Comment:
Risk of exception if `draft.acceptPaymentMethod.option` is undefined at the start of the journey?

Reply:
fixed